tests/Unit/CanonicalJsonTraitTest.php

```diff
@@ -22,6 +22,25 @@ public function testSort(): void
         $unsortedData = json_decode(file_get_contents("$fixturesDirectory/unsorted.json"), true, 512, JSON_THROW_ON_ERROR);
         static::sortKeys($unsortedData);
         $this->assertSame($unsortedData, $sortedData);
+
+        // Indexed arrays should not be sorted in alphabetical order, at any
+        // level.
+        $data = [
+            // This must have at least 10 items, since that will be sorted
+            // alphabetically (which is what we're trying to avoid).
+            'b' => array_fill(0, 20, 'Hello!'),
+            'a' => 'Canonically speaking, I go before b.',
+        ];
```

Comment:
Isn't the other thing about array_is_list is the numeric keys have to be consecutive?

should we test with this also?

```
$not_list = [
          3 => 'three',
          2 => 'two'
        ]
```

This would sorted, right?

Reply:
I think so, yes. I think it can't hurt to add that coverage.